src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go

```diff
@@ -94,6 +96,13 @@ func (g Generator) Generate() error {
 	if err = updateConfig(*siteDir, info); err != nil {
 		return err
 	}
+
+	if g.downloadAll {
+		if err = g.downloadAllMedia(*siteDir, info); err != nil {
```

Comment:
For safety, please assert that `g.downloadAll` is true when `g.downloadMedia` is true.
This avoids confusion where someone might set g.downloadAll to true and `g.downloadMedia` to false.

Reply:
Not sure I understand. `g.downloadAll` is fully independent from `g.dowloadMedia` in the current logic, they can be mixed together or used separately

Reply:
Yeah, as discussed in an earlier comment, I got confused.